java/core/src/main/scala/com/azavea/pdal/Pipeline.scala

```diff
@@ -0,0 +1,27 @@
+package com.azavea.pdal
+
+import ch.jodersky.jni.nativeLoader
+
+@nativeLoader("pdaljni0")
```

Comment:
We should load outside of the JAR resources, since we won't be able to distribute in this way. How will JNI find the required object file on the system? When we were working with GDAL bindings we needed to do something in java options, a la https://github.com/geotrellis/geotrellis-gdal/blob/master/gdal/build.sbt#L18